jinahub/segmenters/VADSpeechSegmenter/tests/integration/test_exec.py

```diff
@@ -0,0 +1,29 @@
+__copyright__ = 'Copyright (c) 2020-2021 Jina AI Limited. All rights reserved.'
+__license__ = 'Apache-2.0'
+
+import os
+
+import pytest
+from jina import Flow, Document, DocumentArray
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
```suggestion
```

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/97/commits/d6a5445f807522ab569712f843bc276be3b46a34) for the change.